store/loomethdb.go

```diff
@@ -146,9 +133,16 @@ type LogParams struct {
 	LogBeforeWriteDump bool
 }
 
+type kvPair struct {
+	key   []byte
+	value []byte
+}
 type LogBatch struct {
-	batch  batch
-	params LogParams
+	parentStore *LoomEthDB
+	size        int
+	params      EthDBLogParams
+	cache       []kvPair
+	lock        sync.RWMutex
```

Comment:
This lock appears to be completely useless, there's only ever going to be a single writer, and the lock isn't used properly in the batch anyway.

Reply:
Yeah I think so but did not want to touch it in the first place. Removed